sdk/cosmos/azure-cosmos/azure/cosmos/aio/_retry_utility_async.py

```diff
@@ -189,6 +193,15 @@ async def ExecuteAsync(client, global_endpoint_manager, function, *args, **kwarg
                 if kwargs['timeout'] <= 0:
                     raise exceptions.CosmosClientTimeoutError()
 
+        except ServiceRequestError:
+            _handle_service_request_retries(request, client, service_response_retry_policy, args)
```

Comment:
L197 should be `_handle_service_request_retries(request, client, service_response_retry_policy, args)`.